omnibus/package-scripts/agent-rpm/prerm

```diff
@@ -115,6 +115,11 @@ remove_sysprobe_secagent_files()
     if [ -d "$INSTALL_DIR/run" ]; then
         rmdir "$INSTALL_DIR/run" 2>/dev/null || true
     fi
+
+    # Remove any unpacked BTF files
+    find "$INSTALL_DIR/embedded/share/system-probe/ebpf/co-re/btf" -name "*.btf*" -type f -delete || true
```

Comment:
Couldn't this be done during the packaging rather than at install time?

Reply:
I think we can only do it here, the BTFs are unpacked during runtime of `system-probe` and `rpm` does not delete the folder because of those files being still present. This would be necessary I think in any case, because the decision of which files to unpack has to be made dynamically on install depending on the running kernel, cannot be done in packaging.

Reply:
That's a very good reason, thanks for the explanation!